support/cas-server-support-jpa-ticket-registry/src/main/java/org/apereo/cas/config/JpaTicketRegistryConfiguration.java

```diff
@@ -164,6 +164,7 @@ public TransactionOperations jpaTicketRegistryTransactionTemplate(
         }
 
         @Bean
+        @ConditionalOnMissingBean(name = TicketRegistry.BEAN_NAME)
```

Comment:
You cannot ever do this. The correct syntax is:

```
@ConditionalOnMissingBean(name = "jpaTicketRegistry")
```

Then you can define your own `jpaTicketRegistry` Bean.